packages/fxa-auth-server/test/client/api.js

```diff
@@ -760,16 +760,6 @@ module.exports = config => {
       );
   };
 
-  ClientApi.prototype.createSubscription = function (planId, paymentToken, headers) {
-    return this.doRequest(
```

Comment:
Moved this down to a block with other subscription-related endpoints and changed it to use `doRequestWithBearerToken`, so it can take the refresh token directly instead of using headers to overwrite the session token behaviour  of `doRequest`.